### PR TITLE
[ADD] sale_order_product_recommendation_product_sold_by_delivery_week

### PR DESCRIPTION
**Pull request by chienandalu**
This module allows to display the weekly sales hint on the sales recommendator.

Depends on 

- [x] https://github.com/OCA/sale-reporting/pull/121

cc @Tecnativa TT31742

ping @sergio-teruel @carlosdauden

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

**Comment by carlosdauden** [comment]
Travis is red

File: sale_order_product_recommendation_product_sold_by_delivery_week/wizard/sale_order_recommendation.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class SaleOrderRecommendationLine(models.TransientModel):
+    _inherit = "sale.order.recommendation.line"
+
+    weekly_sold_delivered_shown = fields.Char(
+        string="Weekly Sold", compute="_compute_weekly_sold_delivered_shown", Trim=False
```
Comment (carlosdauden) [inline sale_order_product_recommendation_product_sold_by_delivery_week/wizard/sale_order_recommendation.py]:
```suggestion
        string="Weekly Sold", compute="_compute_weekly_sold_delivered_shown", trim=False
```

**Comment by carlosdauden** [comment]
Please resolve oca_dependencies.txt conflicts

**Comment by chienandalu** [comment]
> Please resolve oca_dependencies.txt conflicts

Sorry, now is ok

**Comment by carlosdauden** [comment]
Thanks

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**